v22.2/show-create-external-connection.md

```diff
@@ -0,0 +1,70 @@
+---
+title: SHOW CREATE EXTERNAL CONNECTION
+summary: The SHOW CREATE EXTERNAL CONNECTION statement displays all creation statements for active external connections. 
+toc: true
+docs_area: reference.sql
+---
+
+{% include feature-phases/preview.md %}
+
+{% include_cached new-in.html version="v22.2" %} You can use external connections to specify and interact with resources that are external from CockroachDB. When creating an external connection, you define a name for an external connection while passing the provider URI and query parameters. `SHOW CREATE EXTERNAL CONNECTION` displays the connection name and the creation statements for active external connections.
+
+You can also use the following SQL statements to work with external connections:
+
+- [`CREATE EXTERNAL CONNECTION`](create-external-connection.html)
+- [`DROP EXTERNAL CONNECTION`](drop-external-connection.html)
+
+## Required privileges
+
+Users must have the [`admin` role](security-reference/authorization.html#admin-role) to run a `SHOW CREATE EXTERNAL CONNECTION` statement.
```

Comment:
A user doesn't need to be admin to run this command:

- Admin can see all External Connections
- Non-admin can only see the External Connections that they own. Users own the External Connections that they have created.